### ISSUE TITLE
Middle Click on Following blocks the App

### ISSUE THREAD

**Issue by negimaru** [Soundnode/soundnode-app#556]
When the user does a middle click on a following channels the app goes white and stop working.

**Comment by weblancaster** [comment]
Hmm.. that's because open a new blank view/page.. we have blocked the shortcuts.. I need to check that one though.. thanks for letting me know

**Comment by negimaru** [comment]
No, problem, I suppose you are aware but it happens too when the user do the same on a link of the lateral panel(Top 50, Stream, etc... )
